apps/apps.c

```diff
@@ -2633,6 +2636,7 @@ void wait_for_async(SSL *s)
         fds++;
     }
     select(width, (void *)&asyncfds, NULL, NULL, NULL);
+    OPENSSL_free(origfds);
```

Comment:
If the above loop avoids pointer arithmetic, then the origfds would not be needed?
right?  For instance like this:
```
while (numfds > 0) {
  numfds--
  if (width <= (int)fds[numfds])
     width = (int)fds[numfds] + 1;
  openssl_fdset((int)fds[numfds], &asyncfds);
}
select(width, (void *)&asyncfds, NULL, NULL, NULL);
OPENSSL_free(fds);
```